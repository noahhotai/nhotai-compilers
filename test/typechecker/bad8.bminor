i: integer = 3 + 2;