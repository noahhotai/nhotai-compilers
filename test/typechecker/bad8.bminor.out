Program parsed successfully.
Program resolved successfully.
type error: only atomic types allowed to be declared globally
Program failed to typecheck.
